packages/core-db/src/app/ethereum/ethereum-block-processor.ts

```diff
@@ -173,7 +173,13 @@ export class EthereumBlockProcessor {
     }
 
     for (let i = syncStart; i <= blockNumber; i++) {
-      await this.fetchAndDisseminateBlock(provider, i)
+      try {
+        await this.fetchAndDisseminateBlock(provider, i)
+      } catch (e) {
+        logError(log, `Error fetching and disseminating block. Retrying...`, e)
+        i--
+        continue
+      }
```

Comment:
This fixes an issue unrelated with the PR to prevent errors syncing blocks on start-up from causing us to stop syncing. Right now it just retries over and over again.